packages/mui-lab/src/TreeItem/TreeItem.tsx

```diff
@@ -1,6 +1,6 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
What is this for?

Reply:
No idea, I copy what was done in the pickers

The same for why keeping `import PropTypes from 'prop-types';` whereas it's not used